src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```
You can skip the health argument, as it seems it should always start from 100. But it's optional.

Reply:
done